test/utils/shippable/sanity.sh

```diff
@@ -13,18 +13,32 @@ if [ "${install_deps}" != "" ]; then
     apt-get update -qq
     apt-get install shellcheck
 
+    # FIXME REPOMERGE, remove the following
     pip install git+https://github.com/ansible/ansible.git@devel#egg=ansible
```

Comment:
I'm not sure we want to install ansible this way, since we basically end up cloning the repo twice. Once here and then again to get access to the validation script.

We should clone the repo once and use hacking/env-setup to make ansible available. Once the repos are merged, we'll be able to drop the clone (we'll already have the source) and just use hacking/env-setup.